gensrc/thrift/Descriptors.thrift

```diff
@@ -322,6 +322,14 @@ struct THudiTable {
     5: optional list<string> partition_prefixes
 }
 
+struct TJDBCTable {
+    1: optional string jdbc_driver
+    2: optional string jdbc_url
+    3: optional string jdbc_table
+    4: optional string jdbc_user
+    5: optional string jdbc_passwd
+}
+
 // "Union" of all table types.
 struct TTableDescriptor {
```

Comment:
Please only use TJDBCTable. I want to remove `TTableDescriptor` and `TableDescriptor` from our code.

Reply:
> Please only use TJDBCTable. I want to remove `TTableDescriptor` and `TableDescriptor` from our code.

OK, `TJDBCTable` contains all necessary info about external table, but `TTableDescriptor` is stilled passed between FE and BE, It's fine to deal with them uniformly when they are removed. My changes have no additional dependencies on them.